src/main/java/edu/ksu/canvas/attendance/controller/CourseConfigurationController.java

```diff
@@ -86,12 +87,33 @@ public ModelAndView classSetup(@PathVariable String sectionId,
         return page;
     }
 
+    private ModelAndView validateInputAsDouble(String sectionId, CourseConfigurationForm classSetupForm) {
+
+        if (classSetupForm.getGradingOn()){
```

Comment:
All of these fields are defined as Double.  It will always return true. This check doesn't need to be done.